### PR TITLE
remove without check path exist, `rm -f`

### PR DESCRIPTION
**Pull request by luccasmmg**
Fixes https://github.com/nushell/nushell/issues/2531

Old behavior

```
> touch foo bar
> rm -f foo bar | count
2
> rm -f foo bar | count
error: No valid paths
  ┌─ shell:1:1
  │
1 │ rm foo bar
  │ ^^ no valid paths
```

New behavior

```
> touch foo bar
> rm -f foo bar | count
2
> rm -f foo bar | count
0
```
There were much less changes than i thought initially, so i'm kind of apprehensive about.

### CODE REVIEW

File: crates/nu-cli/src/commands/rm.rs [hunk 1]
```diff
@@ -37,6 +38,7 @@ impl WholeStreamCommand for Remove {
                 Some('p'),
             )
             .switch("recursive", "delete subdirectories recursively", Some('r'))
+            .switch("force", "supress error when no file", Some('f'))
```
Comment (thegedge) [inline crates/nu-cli/src/commands/rm.rs:41]: spelling nit: `suppress` (below too)

File: crates/nu-cli/tests/commands/rm.rs [hunk 2]
```diff
@@ -265,3 +264,16 @@ fn remove_files_from_two_parents_up_using_multiple_dots_and_glob() {
...
+
+#[test]
+fn no_errors_if_attempting_to_delete_non_existent_file_with_f_flag() {
+    Playground::setup("rm_test_14", |dirs, _| {
+
+        let actual = nu!(
+            cwd: dirs.root(),
+            "rm -f non_existent_file.txt"
+        );
+        println!("{}", actual.err);
```
Comment (thegedge) [inline crates/nu-cli/tests/commands/rm.rs:275]: I think this line can be removed?

**Review by thegedge — COMMENTED** [review]
Nice! Just a couple of small things and we'll get this one in.

> There were much less changes than i thought initially, so i'm kind of apprehensive about.

I think you got it right. I'll give it a quick little spin before we merge, but I'm not apprehensive 🙂

File: crates/nu-cli/tests/commands/rm.rs (hunk 2 above)
Comment (luccasmmg) [reply crates/nu-cli/tests/commands/rm.rs:275]: ops, forgot to remove that, yes it can

File: crates/nu-cli/src/commands/rm.rs (hunk 1 above)
Comment (luccasmmg) [reply crates/nu-cli/src/commands/rm.rs]: fixed

**Comment by thegedge** [comment]
Works great for me. Thanks, @luccasmmg ! 🎉

**Pull request merged**